src/components/Banner.vue

```diff
@@ -0,0 +1,82 @@
+<template>
+  <section class="section">
+    <div
+      class="padding-small"
+      style="background: #e6f6eb;"
+      v-bind:class="{ change: task.id }"
+    >
+      <div class="columns">
+        <div class="column is-8">
+          <p class="margin-top-normal">
+            If you donate $5 today, and everyone else does the same, we can keep
+            growing our database of freely reusable content. For you. For all of
+            us.
+          </p>
+        </div>
+
+        <div class="column">
+          <button
+            type="submit"
+            class="button is-success is-small position"
+            v-on:click="redirect"
+          >
+            <i
+              class="icon cc-letterheart-filled margin-right-small is-size-5 padding-top-smaller"
+            />Yes, I’ll donate now
+          </button>
+        </div>
+
+        <div class="column">
+          <button
+            class="button is-small position"
+            style="border: none; background: #e6f6eb; color: #008000;"
+            v-on:click="onclick"
+          >
+            Not today
+          </button>
+        </div>
+        <div class="column">
+          <button class="button is-text button_two" v-on:click="onclick">
+            x
+          </button>
+        </div>
+      </div>
+    </div>
+  </section>
+</template>
+<script>
+export default {
+  name: 'Banner',
+  methods: {
+    onclick() {
+      this.task.id = true
+    },
+    redirect() {
+      window.location.href =
+        'https://www.classy.org/give/297881/#!/donation/checkout'
+    },
```

Comment:
Is there a reason to do this instead of using a `<a href=""></a>` tag? I think the anchor link would be better unless there is a good reason.

Reply:
Updated @zackkrida as u said😊